submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/service/SubmissionService.kt

```diff
@@ -38,11 +40,17 @@ class SubmissionService(
         submissionRepository.expireSubmission(accNo)
     }
 
-    fun submit(
-        submission: Submission,
-        user: SecurityUser,
-        files: UserSource
-    ) = submitter.submit(ExtendedSubmission(submission, asUser(user)), files, persistenceContext)
+    fun submit(submission: Submission, user: SecurityUser, files: UserSource) =
+        submitter.submit(ExtendedSubmission(submission, asUser(user)), files, persistenceContext)
+
+    fun getFormat(file: File): SubFormat {
+        return when (file.extension) {
```

Comment:
Maybe these could be extension methods for the `File` class like: `isTsv()`, `isXml()` or `isJson()`. Something like this:

https://github.com/EBIBioStudies/biostudies-backend-services/pull/94/files#diff-cbe783e2044ed31f5f92ff6134f0a5d8R9